build/release/Makefile

```diff
@@ -45,15 +45,7 @@ else
 DOCS_VERSION_ALIAS :=
 endif
 
-DOCS_DIR ?= $(ROOT_DIR)/Documentation
-DOCS_WORK_DIR := $(WORK_DIR)/rook.github.io
-DOCS_VERSION_DIR := $(DOCS_WORK_DIR)/docs/rook/$(DOCS_VERSION)
```

Comment:
What haooened to these three variables?

Reply:
These three variables are not used anywhere in the Makefiles. I removed them as it was confusing changing these and nothing changing when building/deploying the docs.